legion/engine/physics/colliders/convexcollider.cpp

```diff
@@ -685,7 +628,29 @@ namespace legion::physics
         AssertEdgeValidity();
         //log::debug("-> Finish ConstructConvexHullWithMesh ----------------------------------");
     }
-    
+
+    void ConvexCollider::populateVertexListWithHalfEdges()
+    {
+        auto& verticesVec = vertices;
+
+        int reserveSize = halfEdgeFaces.size() * 3;
```

Comment:
use size_t instead of int - it can't go negative and vector uses size_t for reserve() and size() as well so no extra (implicit) casts need to be made

Reply:
the reserve is to account for the minimal amount of edges a single face can have (3). But I agree it should be const.